presets/react/storybook/ssr/src/utils.jsx

```diff
@@ -0,0 +1,17 @@
+import qs from 'qs';
+
+export const getStories = () => {
+  const stories = require.context(`${PKG_PATH}/src`, true, /\.stories\.jsx$/);
+  return stories.keys().map(stories);
+};
+
+export const getUrlForStory = ({ storyId, viewMode }) => {
+  const { path, selectedKind, selectedStory, ...rest } = qs.parse(document.location.search, {
+    ignoreQueryPrefix: true,
```

Comment:
slicing would be shorter code

Reply:
used nebenan-helpers instead, should be better now